src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -44,16 +82,56 @@ public static void PrintProcessStatus(IConsole console)
                     {
                         continue;
                     }
-
                     try
                     {
-                        sb.Append($"{process.Id, 10} {process.ProcessName, -10} {process.MainModule.FileName}\n");
+                        if(verbose)
+                        {
+                            int iDLength = 10;
+                            int nameLength = 10;
+                            int fileLength = 25;
+                            int cmdLength = 55;
+                            String cmdLineArgs = GetArgs(process);
+                            cmdLineArgs = cmdLineArgs == process.MainModule?.FileName ? "" : cmdLineArgs;
+                            string fileName = process.MainModule?.FileName ?? "";
+                            string[] cmdList = cmdLineArgs.Split(" ");
+                            foreach(string str in cmdList)
+                            {
+                                string splitChar = "";
+                                if (str.Contains("/"))
+                                {
+                                    splitChar = "/";
+                                    
+                                }
+                                else if(str.Contains("\\"))
+                                {
+                                    splitChar = "\\";
+                                }
+                                else
+                                {
+                                    continue;
+                                }
+                                string[] filePath = str.Split(splitChar);
+                                fileName = filePath.LastOrDefault();
+                                break;
+                            }
+                            string processID = MakeFixedWidth(process.Id.ToString(), iDLength);
+                            string processName = MakeFixedWidth(process.ProcessName, nameLength);
+                            fileName = MakeFixedWidth(fileName, fileLength);
+                            cmdLineArgs = MakeFixedWidth(cmdLineArgs, cmdLength, true);
+                            string toAppend = $"{processID, 10} {processName, -10} {fileName, -25} {cmdLineArgs, -55}\n";
+                            sb.Append(toAppend);
```

Comment:
Rather than have these all be arbitrary fixed widths, I think it makes sense to determine the fixed width based on the longest value of each property. The information won't be useful if the path or sommand line arguments are longer than these character counts.

Instead, you could build up an array of tuples that contain the information and determine the column widths using a `Max` operation on each property.